src/main/java/com/github/dockerjava/core/command/BuildImageCmdImpl.java

```diff
@@ -84,7 +84,11 @@ public BuildImageCmdImpl(BuildImageCmd.Exec exec, InputStream tarInputStream) {
 
     @Override
     public String getTag() {
-        return tag;
+        if (tags == null || tags.isEmpty()) {
+            return null;
+        }
+        // return first tag to be backward compatible
+        return tags.iterator().next();
```

Comment:
if i right understand `witTag(String)` is already existing and we can Deprecate it and remove in future and add new Set?